### ISSUE TITLE
Add `output.files()` method?

### ISSUE THREAD

**Issue by rwjblue** [broccolijs/broccoli-test-helper#5]
I commonly have to write the following code to ensure that the list of files is as expected:

```js
it('...', co.wrap(function* () {
  // ...snip...
  output = createBuilder(someTree);

  yield output.build();

  let files = walkSync(output.path(), { directories: false });
  expect(files).to.deep.equal(['some.txt', 'list.pdf', 'of.png', 'files.gif']);
});
```

What are your thoughts on making this a tad bit easier and exposing a `files` method?

**Comment by krisselden** [comment]
this seems very similar to a deep equal of the tree

**Comment by krisselden** [comment]
I'm ok with it just curious

**Comment by rwjblue** [comment]
@krisselden - Right, but I don't want to assert on the contents (just the files).  I have other tests around the contents..

**Comment by krisselden** [comment]
Seems ok, I would ensure it returns them in a consistent order.

**Comment by krisselden** [comment]
This is fixed by #9

**Comment by krisselden** [comment]
You can now do `assert.deepEqual(output.readDir(), expectedArrayOfFiles)`
